### ISSUE TITLE
Alert notification message not spoken in IE10 metro/modern browser.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3325]
**Reported by HariC on 2013-07-02 14:20**
Unlike in desktop version of IE10, NVDA fails to speak alert notification message in metro/modern version of IE10.

**Comment by leonardder** [comment]
IE10 metro is replaced by edge in Windows 10, and I belief alerts are working in Edge just fine. Closing as won't fix.
